src/utils/math.ts

```diff
@@ -141,7 +142,7 @@ export function forceFillColumnWidths(
     contentWidth = getContentWidth(allColumns);
     remainingWidth = expectedWidth - contentWidth;
     removeProcessedColumns(columnsToResize, columnsProcessed);
-  } while (remainingWidth > 0 && columnsToResize.length !== 0);
+  } while (Math.abs(remainingWidth) > remainingWidthLimit && columnsToResize.length !== 0);
```

Comment:
Why the Math.abs? In our case expectedWidth is negative (seems like it's counted when column mode is forced & scrollbarV this is counted as innerWidth - scrollbarHelper.width) and that will result to an infinite loop.

Reply:
Yeah this caused an infinite loop with my usage so I've reverted it locally to ensure I can still use this library